src/main/java/net/minecraftforge/client/model/ModelLoader.java

```diff
@@ -670,7 +670,16 @@ public WeightedRandomModel(ResourceLocation parent, VariantList variants) throws
                 textures.addAll(model.getTextures()); // Kick this, just in case.
 
                 models.add(model);
-                builder.add(Pair.of(model, new ModelStateComposition(v.getState(), model.getDefaultState())));
+
+                IModelState modelDefaultState = model.getDefaultState();
```

Comment:
Can use something like `Preconditions.checkNotNull` from Guava here.

Reply:
this is just a one-line change

Reply:
My comment was referring to an earlier version of the code that didn't use `Preconditions`.

Reply:
I mean the whole thing can be squeezed in one line